src/extension.ts

```diff
@@ -1188,6 +1188,7 @@ export async function activate(context: vscode.ExtensionContext) {
 
   registerIDFCommand("espIdf.createIdfTerminal", createIdfTerminal);
 
+  registerIDFCommand("espIdf.flashDFU", flash(false));
```

Comment:
While trying to launch dev mode, I've received this error here:

```
ERROR in /Users/brainignacio/workspace/vscode-esp-idf-extension/src/extension.ts
    [tsl] ERROR in /Users/brainignacio/workspace/vscode-esp-idf-extension/src/extension.ts(1191,41)
          TS2345: Argument of type 'void' is not assignable to parameter of type '(...args: any[]) => any'.
```

This is because `flash(false)` is passing the result of the function not the function itself, which is the intended goal here.